recipe/meta.yaml

```diff
@@ -100,10 +100,9 @@ build:
   # match python.org compiler standard
   skip: True            # [win and int(float(vc)) < 14]
 # Would like to be able to append to the calculated build string here, ping @msarahan, is this possible?
-# Also this messes up hash generation so package ends up without any :-(
-{% if linkage_nature != '' or debug != '' %}
-  string: {{ version }}{{ linkage_nature }}{{ debug }}
-{% endif %}
+  string: h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}{{ linkage_nature }}{{ debug }}_cpython
```

Comment:
@mingwandroid, I've kept your changes here.

Reply:
Yes, I saw. And I have kept yours in my WIP changes, is, I suppose what I meant to say.